src/vars/images.yml

```diff
@@ -3,6 +3,9 @@ candlepin_container_image: quay.io/foreman/candlepin
 candlepin_container_tag: "foreman-{{ container_tag_stream }}"
 foreman_container_image: quay.io/foreman/foreman
 foreman_container_tag: "{{ container_tag_stream }}"
+foreman_proxy_container_image: "quay.io/foreman/foreman-proxy"
+foreman_proxy_container_tag: "{{ container_tag_stream }}"
```

Comment:
I wanted to say "please add this to the `images` list below" but given it's an optional feature, I am not that sure.

@ehelms @ekohl opinions?

Reply:
#188 is updated and makes "foreman-proxy" a feature, have a look :)